service/src/main/java/tds/session/repositories/SessionRepository.java

```diff
@@ -18,6 +19,14 @@
     Optional<Session> findSessionById(final UUID id);
```

Comment:
Can we remove this method?  I don't see the point of it with the other one since the underlying code uses the exact same query.  Actually, is this even used anymore after your changes?